torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -39,17 +39,22 @@ static std::string spmd_device_str = "SPMD:0";
 
 // Initializes a distributed runtime client if dist_service_addr is specified
 std::shared_ptr<xla::DistributedRuntimeClient>
-MaybeInitializeDistributedRuntimeClient(int local_rank,
-                                        std::string dist_service_addr) {
+MaybeInitializeDistributedRuntimeClient(int local_rank) {
   std::shared_ptr<xla::DistributedRuntimeClient> client;
-  if (!dist_service_addr.empty()) {
-    xla::DistributedRuntimeClient::Options options;
-    /* TODO(jonbolin): Use global rank for multi-host setup */
-    options.node_id = local_rank;
-    client = xla::GetDistributedRuntimeClient(dist_service_addr, options);
-    XLA_CHECK(client->Connect().ok())
-        << "Failed to initialize distributed runtime client";
+  int global_world_size = sys_util::GetEnvInt("WORLD_SIZE", 1);
+  if (global_world_size < 2) {
+    return std::move(client);
   }
+  std::string master_addr = sys_util::GetEnvString("MASTER_ADDR", "127.0.0.1");
+  std::string port = runtime::sys_util::GetEnvString("COORDINATOR_PORT", "8547");
+  std::string dist_service_addr = master_addr+":"+port ;
```

Comment:
nit: can you use `absl::StrJoin` instead? https://abseil.io/docs/cpp/guides/strings#abslstrjoin-for-joining-elements-within-a-string

Reply:
Thanks!